src/sentry/south_migrations/0369_correct_groupsnooze_windows.py

```diff
@@ -0,0 +1,1052 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        GroupSnooze = orm.GroupSnooze
+
+        snoozes = GroupSnooze.objects.filter(
+            models.Q(window__isnull=False) | models.Q(user_window__isnull=False),
+        )
+        for snooze in RangeQuerySetWrapperWithProgressBar(snoozes):
+            if snooze.window is not None:
+                GroupSnooze.objects.filter(id=snooze.id).update(window=snooze.window * 60)
+            if snooze.user_window is not None:
+                GroupSnooze.objects.filter(id=snooze.id).update(user_window=snooze.user_window * 60)
```

Comment:
Can't this be updated with an `F` object instead? Or will that not work with nulls?